gateway-service-impl/src/test/java/org/hypertrace/gateway/service/common/AttributeCacheKeyTest.java

```diff
@@ -48,7 +49,8 @@ public void testAttributeCacheKey() {
   private RequestContext buildContext(String tenantId, Map<String, String> headers) {
     org.hypertrace.core.grpcutils.context.RequestContext grpcContext =
         org.hypertrace.core.grpcutils.context.RequestContext.forTenantId(tenantId);
-    headers.forEach(grpcContext::add);
+    grpcContext.removeHeader(RequestContextConstants.REQUEST_ID_HEADER_KEY);
```

Comment:
grpc context generate a unique rest id from a while back - removing it for the purpose of this test since comparisons are done across headers.